invisible_cities/reco/krmap_functions_test.py

```diff
@@ -104,3 +105,62 @@ def test_transform_parameters(a, b):
     npt.assert_allclose(transformed_par, [  E0_expected,   lt_expected])
     npt.assert_allclose(transformed_err, [s_E0_expected, s_lt_expected])
     assert np.isclose(transformed_cov, cov_expected)
+
+
+@given(integers(min_value = 0, max_value = 1e10),
+       integers(min_value = 0, max_value = 1e10),
+       arrays  (dtype     = np.int64, shape = (2,),
+                elements  = integers(min_value = 1,
+                                     max_value = 1e4)))
+def test_get_number_of_bins_performance_default_value(nevents, thr, n_bins):
+    bins = krf.get_number_of_bins(nevents, thr, n_bins)
+
+    npt.assert_array_equal(bins, n_bins)
```

Comment:
Simplify this test: fix a value for `nevents` and `thr` and parametrize `n_bins` with a few values, instead of letting hypothesis generate a zillion values.